charts/values/values.common.yaml

```diff
@@ -0,0 +1,41 @@
+# Configure the images to use when generating manifests.
+node:
+  image: docker.io/calico/node
+  tag: master
```

Comment:
>There's a remaining question of how we do the tiny bits of more "dynamic" templating - e.g., setting the image versions in the manifests - that used to by handled by dynamically building values.yaml. But perhaps that can be handled more simply via something like sed. TBD.


I think the question of how we set the correct image versions in the manifest is pretty tricky now that the source of truth has been split out to another directory, so i'm eager to hear ideas. Honestly the current system of "script which parses _data/versions.yml and renders a templated versions" might still be the way to go, just not in ruby anymore. Similar to the gen-versions in tigera-operator

Reply:
@ozdanborne the problem is that I don't want any coupling with _data/versions.yaml, since that's a co-dependence on the docs.

Agree what we have in the PR right now is not ideal, still working through options for that. Maybe we'll need to have that co-dependence but would rather avoid it especially as we move towards pulling the docs into a new repo, and eventually the tigera-operator chart out of this repo as well.

Reply:
I feel it's not a codependence on the docs, but rather a codependence on versions.yml, which basically is a schema shared between our repos like hashreleases and operator.

Reply:
Really all we need at this point is:

```
calico: vX.Y.Z
operator: vA.B.C
```

Could probably make this a lot simpler by just changing our charts to follow that model, and then it's just literally two strings to modify.

Reply:
Pushed a commit that uses a CALICO_VERSION and OPERATOR_VERSION env var to allow configuration of this on the fly (e.g., for hashreleases), but we still default to the value from values.yaml.

Hoping this will make it simpler to manage.